build/crds/build-crds.sh

```diff
@@ -46,15 +46,6 @@ copy_ob_obc_crds() {
 generating_crds_v1() {
   echo "Generating v1 in crds.yaml"
   "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/ceph.rook.io/v1" output:crd:artifacts:config="$OLM_CATALOG_DIR"
-  $YQ_BIN_PATH w -i cluster/olm/ceph/deploy/crds/ceph.rook.io_cephclusters.yaml spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.mon.properties.stretchCluster.properties.zones.items.properties.volumeClaimTemplate.properties.metadata.x-kubernetes-preserve-unknown-fields true
```

Comment:
What if we kept a `yq` command here to keep consistent spacing? Then if we need another `yq` command in the future we won't again have the whitespace change.

Reply:
+1 we could just parse with `yq` to get the formatting so we don't rollback the formatting each time.

Reply:
🤷‍♀️  Sure. I don't love it, but I suppose it doesn't hurt. Mostly I'd rather not rely on `yq` parsing as much as we can help it in the future.